src/views/TenderTable/TenderTable.vue

```diff
@@ -0,0 +1,188 @@
+      count: 5,
+      page: 1,
+      tenderCount: 0,
+      offers: [],
+      tenders: []
+    }
+  },
+  async beforeMount() {
+    this.tenders = await fetch(API_BASE_URL+ '/tenders/me?page=' + (this.page-1)+'&count='+this.count, {
+      method: 'GET',
+      node: 'cors',
+      cache: 'no-cache',
+      credentials: 'same-origin',
+      headers: getAuthenticatedHeaders(),
+    }).then((response) => {
+          if (response.ok) {
+            return response.json()
+          } else if(response.status === 401 || response.status === 403){
+            logout();
+
+          }
+        }
+    )
+        .then(response => {
+          return response;
+        });
+    this.tenderCount = await fetch(API_BASE_URL+'/tenders/me/count', {
+      method: 'GET',
+          node: 'cors',
+          cache: 'no-cache',
+          credentials: 'same-origin',
+          headers: getAuthenticatedHeaders(),
+    }).then((response) => {
+          if (response.ok) {
+            return response.json()
+          } else if(response.status === 401 || response.status === 403){
+            logout();
+          }
+        }
+    )
+        .then(response => {
+          return response;
+        });
+
+    this.offers = await fetch(API_BASE_URL+'/tenders/me/offers?page=0&count=100', {
+      method: 'GET',
+      node: 'cors',
+      cache: 'no-cache',
+      credentials: 'same-origin',
+      headers: getAuthenticatedHeaders(),
+    }).then((response) => {
+          if (response.ok) {
+            return response.json()
+          } else if(response.status === 401 || response.status === 403){
+            logout();
+          }
+        }
+    )
+        .then(response => {
+          return response;
+        });
+  },
+
+  methods: {
+
+    async pageChange(){
+      this.tenders = await fetch(API_BASE_URL+'/tenders/me?page=' + (this.page-1)+'&count='+this.count, {
+        method: 'GET',
+        node: 'cors',
+        cache: 'no-cache',
+        credentials: 'same-origin',
+        headers: getAuthenticatedHeaders(),
+      }).then((response) => {
+            if (response.ok) {
+              return response.json()
+            } else if(response.status === 401 || response.status === 403){
+              logout();
+            }
+          }
+      )
+          .then(response => {
+            return response;
+          });
+
+      this.offers = await fetch(API_BASE_URL+'/tenders/me/offers?page=0&count=100', {
+        method: 'GET',
+        node: 'cors',
+        cache: 'no-cache',
+        credentials: 'same-origin',
+        headers: getAuthenticatedHeaders(),
+      }).then((response) => {
+            if (response.ok) {
+              return response.json()
+            } else if(response.status === 401 || response.status === 403){
+              logout();
+            }
+          }
+      )
+          .then(response => {
+            return response;
+          });
+    },
+    formatDate(miliseconds) {
+      let d = new Date(miliseconds);
```

Comment:
better to use smth like this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString